srcpkgs/flrig/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'flrig'
+pkgname=flrig
+version=1.4.7
+revision=1
+build_style=gnu-configure
+makedepends="fltk-devel"
+depends="fltk"
+short_desc="Rig control program, compatible with fldigi"
+maintainer="Antonio Gurgel <antonio@goorzhel.com>"
+license="GPL-3.0-or-later"
```

Comment:
license is unclear, there's a COPYING file that is GPL 2, but src/main.cxx says GPL 3+. Other distros use a [variety of values](https://repology.org/project/flrig/information). Could you investigate this?

Reply:
The COPYING is v3:
```
/tmp/fldigi-4.1.23 ❯ head -2 COPYING
                    GNU GENERAL PUBLIC LICENSE
                       Version 3, 29 June 2007
```
But it ships mbedtls in `src/mbedtls` under GPL v2. Thoughts?

Reply:
weird, that's not what [this shows](https://sourceforge.net/p/fldigi/flrig/ci/master/tree/COPYING)

Reply:
I've emailed the author.

Reply:
He wrote:
> GPL-v3 or later is the intended license for all of the NBEMS suite applications including
>  * fldigi / flarq
>  * flrig
>  * flcluster
>  * flmsg
>  * flamp
>  * fllog
>  * flwrap
>  * flwkey
>  * flnet
>  * flaa
>  * flxmlrpc
>  * flnano
>  * linsim
>  * comptty
>  * comptext
>  * pistats

Reply:
ok, sounds good